Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM debian:latest
+FROM python:buster
+
+RUN mkdir /autoreduce
+WORKDIR /autoreduce
+
+RUN apt-get update && apt-get install -y sudo && apt-get install -y software-properties-common
+# RUN useradd -m docker && echo "docker:docker" | chpasswd && adduser docker sudo
+
+ADD requirements.txt .
+
+RUN python -m pip install -r requirements.txt
+
+ADD ./build ./build
+ADD ./setup.py .
+# ADD test_credentials.log ./utils/test_credentials.ini
```

Comment:
I guess all these commented out hacks could be removed :D

Reply:
Yeah I need to clean-up the files and add a proper readme, as well as build scripts to automate the builds. 

I think ultimately there will be 3 Dockerfiles for DB/Queues/WebApp, we can probably base all 3 on the WebApp one though, and then figure out how to compose it so that you can just do `docker-compose run autoreduction.yml` (or something like this 😄) and get the whole thing running